docs-ref-services/sql.md

```diff
@@ -19,46 +19,67 @@ ms.service:
 
 Work with data stored in  [Azure SQL Database](https://docs.microsoft.com/azure/sql-database/sql-database-technical-overview) from Python with the Microsoft ODBC driver. 
 
-The management libraries provide an interface to create, manage, and scale Azure SQL Database deployments from your Python code. Set up and manage databases in [elastic pools](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-elastic-pool) to share resources and configure databases across multiple regions from your code.
+## Client ODBC driver and pyodbc
+Connect to the Azure SQL Database from your applications using the [Microsoft ODBC driver and pyodbc](https://docs.microsoft.com/azure/sql-database/sql-database-connect-query-python#install-the-python-and-database-communication-libraries).
 
-## Install the libraries
+### Example
 
-### ODBC driver 
-[Instructions](https://docs.microsoft.com/azure/sql-database/sql-database-connect-query-python#install-the-python-and-database-communication-libraries) for downloading the Microsoft ODBC driver.
+Connect to a SQL database and select all records in a table.
 
-### Management
+```python
+connection_string = 'Driver={ODBC Driver 13 for SQL Server};Server=tcp:samplesqlserver123.database.windows.net,1433;Database=samplesqldb;Uid=mysecretname@samplesqlserver123;Pwd={testpassword};Encrypt=yes;Timeout=30;'
+
+cnxn = pyodbc.connect(connection_string)
+cursor = cnxn.cursor()
+selectsql = "SELECT * FROM SALES"
+cursor.execute(selectsql)
+```
+
+## Management API
+
+Create and manage Azure SQL Database resources in your subscription with the management API. 
 
 ```bash
 pip install azure-mgmt-sql
 ```
+> [!div class="nextstepaction"]
+> [Explore the Management APIs](/python/api/azure.mgmt.sql)
 
-## ODBC Example
+### Example
 
-Connect to a Azure SQL database and select all records in the sales table.
+Create a SQL Database resource and restrict access to a range of IP addresses using a firewall rule.
 
 ```python
-server = SERVER_NAME+'.database.windows.net'
-database = DATABASE_NAME
-username = USER_NAME
-password = PASSWORD
-driver = '{ODBC Driver 13 for SQL Server}'
-
-cnxn = pyodbc.connect(
-    'DRIVER=' + driver + ';PORT=1433;SERVER=' + server + ';PORT=1443;DATABASE=' + database + ';UID=' + username + ';PWD=' + password)
-cursor = cnxn.cursor()
-selectsql = "SELECT * FROM SALES"
-cursor.execute(selectsql)
+# Create a SQL server
+server = sql_client.servers.create_or_update(
+    sqlresourcegroup,
+    sqldb123,
+    {
+        'location': LOCATION,
+        'version': '12.0', # Required for create
+        'administrator_login': USER_NAME, # Required for create
+        'administrator_login_password': PASSWORD # Required for create
+    }
+)
+
+# Open access to this server for IPs
+firewall_rule = sql_client.firewall_rules.create_or_update(
+    sqlresourcegroup,
+    sqldb123,
+    "firewall_rule_name_123.123.123.123",
```

Comment:
move constants to the top of the code block or explain in comments what the code does